packages/terra-heading/src/Heading.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Heading.scss';
+
+const propTypes = {
+  /**
+   * Needs descriptions
+   */
+  children: PropTypes.node.isRequired,
```

Comment:
You have a lot of good docs on your github markdown pages. Given that this is used to generate the props table, perhaps it wouldn't be a bad idea to include links to the github documentation as part of the prop documentation?